### ISSUE TITLE
Non-static method _getSectionSources() cannot be called statically

### ISSUE THREAD

**Issue by Tam** [verbb/formie#676]
**Description**
```
Error: Non-static method verbb\formie\helpers\RichTextHelper::_getSectionSources() cannot be called statically in /var/www/html/vendor/verbb/formie/src/helpers/RichTextHelper.php:125
Stack trace:
#0 /var/www/html/vendor/verbb/formie/src/helpers/RichTextHelper.php(41): verbb\formie\helpers\RichTextHelper::_getLinkOptions()
#1 /var/www/html/vendor/verbb/formie/src/services/Notifications.php(568): verbb\formie\helpers\RichTextHelper::getRichTextConfig('notifications.c...')
#2 /var/www/html/vendor/verbb/formie/src/services/Notifications.php(469): verbb\formie\services\Notifications->defineContentSchema()
#3 /var/www/html/vendor/verbb/formie/src/controllers/FormsController.php(527): verbb\formie\services\Notifications->getNotificationsSchema()
#4 /var/www/html/vendor/verbb/formie/src/controllers/FormsController.php(97): verbb\formie\controllers\FormsController->_prepareVariableArray(Array)
#5 [internal function]: verbb\formie\controllers\FormsController->actionEdit(457, NULL, NULL)
#6 /var/www/html/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#7 /var/www/html/vendor/yiisoft/yii2/base/Controller.php(181): yii\base\InlineAction->runWithParams(Array)
#8 /var/www/html/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction('edit', Array)
#9 /var/www/html/vendor/craftcms/cms/src/web/Application.php(287): yii\base\Module->runAction('formie/forms/ed...', Array)
#10 /var/www/html/vendor/yiisoft/yii2/web/Application.php(104): craft\web\Application->runAction('formie/forms/ed...', Array)
#11 /var/www/html/vendor/craftcms/cms/src/web/Application.php(272): yii\web\Application->handleRequest(Object(craft\web\Request))
#12 /var/www/html/vendor/yiisoft/yii2/base/Application.php(392): craft\web\Application->handleRequest(Object(craft\web\Request))
#13 /var/www/html/web/index.php(33): yii\base\Application->run()
#14 {main}
```

**Steps to reproduce**
1. Occurs when editing a form

**Additional info**
- Plugin version: 1.5.0
- Craft version: 3.7.24
- Multi-site: No

**Additional context**
Happened after updating from 1.4.27 to 1.5.0

**Comment by engram-design** [comment]
Looks like this is a PHP 8 issue. Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.5.0",
  "...": "..."
}
```

Then run `composer update`.

**Comment by Tam** [comment]
Prefect, ta!

**Comment by juddlyon** [comment]
I can confirm the same issue (Craft CMS 3.7.25.1 & Formie 1.5.0). Updating `composer.json` to `dev-craft-3 as 1.5.0` fixed the issue. Thanks @engram-design!

**Comment by engram-design** [comment]
Fixed in [1.5.1](https://github.com/verbb/formie/releases/tag/1.5.1)
